qutebrowser/config/configdata.py

```diff
@@ -1239,6 +1239,8 @@ def data(readonly=False):
         ('stop', ['<Ctrl-s>']),
         ('print', ['<Ctrl-Alt-p>']),
         ('open qute:settings', ['Ss']),
+        ('select-follow', ['<Return>']),
```

Comment:
You're right, `<Return>` does not work anymore in normal mode. I made a command because I did not no where to listen for the `<Return>` key, but it does not make sense to have a command for this feature, in my opinion. To make it work, where should I listen for this key?